src/Host/Client/Test/DataInspection/ValuesTest.cs

```diff
@@ -31,13 +31,14 @@ public class ValuesTest : IAsyncLifetime {
         private const REvaluationResultProperties AllFields = unchecked((REvaluationResultProperties)~0);
 
         private readonly MethodInfo _testMethod;
+        private readonly IRHostBrokerConnector _brokerConnector = new RHostBrokerConnector(name: nameof(ValuesTest));
```

Comment:
In constructor. 2 reasons: better stack trace in case of exception and explicit order of instantiation

---
In reply to: [75011847](https://github.com/Microsoft/RTVS/pull/2173#discussion_r75011847) [](ancestors = 75011847,74972244)